JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -0,0 +1,241 @@
+import { NgModule, Component, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import {
+  DxToolbarModule,
+  DxSelectBoxModule,
+  DxTemplateModule,
+  DxResizableModule,
+  DxDropDownButtonModule,
+  DxButtonGroupModule,
+  DxCheckBoxModule,
+  DxRadioGroupModule,
+} from 'devextreme-angular';
+
+import DataSource from 'devextreme/data/data_source';
+import notify from 'devextreme/ui/notify';
+import {
+  FontFamilies,
+  FontSize,
+  FontStyle,
+  LineHeight,
+  ListType,
+  Service,
+  TextAlign,
+  TextAlignExtended,
+} from './app.service';
+
+if (!/localhost/.test(document.location.host)) {
```

Comment:
I'm not sure that it is good practice to enable/disable production mode in components by hostname.

Usually, the production mode is controlled from the main.ts project file by the environment.
[Link to the example Angular project](https://github.com/btoorenburg/angular-default-project/blob/master/src/main.ts)

But in all our Demos do it here.
Maybe we've some reasons for this or we need to discuss it and change all our Angular demos.

Reply:
The discussion is not over. We decided to use the existing code style.